examples_tests/06.MeshLoaders/main.cpp

```diff
@@ -180,13 +180,16 @@ int main()
         gpumesh = (*gpu_array)[0];
     }
 
-    core::smart_refctd_ptr<video::IGPUGraphicsPipeline> gpuGraphicsPipeline;
+    std::vector<core::smart_refctd_ptr<video::IGPUGraphicsPipeline>> gpuGraphicsPipelines;
```

Comment:
make a `map<RenderpassIndependent,GraphicsPipeline>`

Reply:
corrected in following commit